packages/api/src/external/fhir/document/draft-update-document-reference.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Attachment,
+  DocumentReference,
+  DocumentReferenceContent,
+  Organization,
+} from "@medplum/fhirtypes";
+import { createFHIRSubject } from ".";
+import BadRequestError from "../../../errors/bad-request";
+import { OrganizationModel } from "../../../models/medical/organization";
+import { metriportDataSourceExtension } from "../shared/extensions/metriport";
+
+export function cxDocRefCheck(docRef: DocumentReference) {
```

Comment:
why `cx` on the name? IIUIC it could be `docRefCheck`, as its checking a document reference, right?